modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -237,4 +237,89 @@ describe('Field Checkbox Multiple', () => {
 
 		expect(handleFieldEdited).toHaveBeenCalled();
 	});
+
+	it('checks the value if there is a value', () => {
+		const {getByLabelText} = render(
+			<CheckboxMultipleWithProvider
+				options={[
+					{
+						label: 'Option 1',
+						value: 'option1',
+					},
+					{
+						label: 'Option 2',
+						value: 'option2',
+					},
+					{
+						label: 'Option 3',
+						value: 'option3',
+					},
+				]}
+				predefinedValue={['option1', 'option2']}
+				spritemap={spritemap}
+				value={['option3']}
+			/>
+		);
+
+		expect(getByLabelText('Option 1').checked).toBe(false);
+		expect(getByLabelText('Option 2').checked).toBe(false);
+		expect(getByLabelText('Option 3').checked).toBe(true);
```

Comment:
Prefer assertions methods from [jest-dom](https://github.com/testing-library/jest-dom), because they provide a more accurate error message in case of failure
```suggestion
		expect(getByLabelText('Option 1')).not.toBechecked();
		expect(getByLabelText('Option 2')).not.toBeChecked();
		expect(getByLabelText('Option 3')).toBeChecked();
```

Reply:
I think this test has no `jest-dom` imported yet, so you need to add this at the imports:

```javascript
import '@testing-library/jest-dom';
```
https://github.com/testing-library/jest-dom#usage